### DISCUSSION TITLE
Trying to patch using cmd:HELP

### DISCUSSION THREAD

**Discussion by Gsparsh22** [ReVanced/revanced-discussions#1112] [Question] [answered]
Trying to patch revanced using cmd, please help with the command line:

java -jar rev-cli.jar patch -p rev-patch.rvp --mount rev-integration.apk -o revanced_19.46.42.apk youtube_19.46.42.apk

Error: Unmatched argument at index 7: 'youtube_19.46.42.apk'

What should be the argument used along the apk to be patched.

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
`Rev-integration.apk` should not be used

**Reply by Gsparsh22** [reply]
Thanks man!!! Really appreciate it. But why is that so?

**Reply by LisoUseInAIKyrios** [reply]
The patcher changed recently and there is no integrations file anymore.
